src/lib/fountain/test/FountainMetadataTest.cpp

```diff
@@ -26,6 +26,14 @@ TEST_CASE( "FountainMetadataTest/testFromBytes.Short", "[unit]" )
 	assertEquals( 0x070809, inmd.file_size() );
 }
 
+TEST_CASE( "FountainMetadataTest/testFromBytes.BigFile", "[unit]" )
+{
+	std::string buff = {(char)0x81, 7, 8, 9};
```

Comment:
Added a few tests for my sanity. This is technically redundant, but it's nice to see it declared explicitly.